File: src/Java/BehavioralPatterns/Mediator/ChainReaction/src/FontEditor.java
public class FontEditor extends Participant {
    public FontEditor(Mediator mediator) {
        super(mediator);
    }

    public void enable() {
        System.out.println("FontEditor is enabled.");
    }

    public void disable() {
        System.out.println("FontEditor is disabled.");
    }
}

File: src/JavaScript/BehavioralPatterns/Mediator/ChainReaction/fontEditor.js

const Participant = require("./participant");

class FontEditor extends Participant {
    constructor(mediator) {
        super(mediator);
    }

    enable() {
        console.log("FontEditor is enabled.");
    }

    disable() {
        console.log("FontEditor is disabled.");
    }
}

module.exports = FontEditor;